### DISCUSSION TITLE
Changing the timezone

### DISCUSSION THREAD

**Discussion by yula-21** [Security-Onion-Solutions/securityonion#7756] [Unsupported Versions] [answered]
Hi,
I want to change the timezone as it is currently utc and I’ve ran into some issues.
Since all the dockers work in utc it causes problems to change the host and it keeps changing back.
The web interface gets the timezone from the browser and its correct but I need to change the machine’s tz as well.

Does anyone have a solution?

**Comment by dougburks** [comment, marked as answer]
UTC is the correct timezone and you should avoid changing it:
https://docs.securityonion.net/en/2.3/timezones.html
